File: app/helloworld_compatactivity/src/main/java/org/example/application/MainActivity.java
package org.example.application;


import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;
import android.view.Window;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        this.requestWindowFeature(Window.FEATURE_NO_TITLE);
        TextView textView = new TextView(this);
        textView.setText("Hello world");
        setContentView(textView);
    }
}

File: app/helloworld_kotlin/src/main/java/org/example/application/MainActivity.kt
package org.example.application

import android.app.Activity
import android.os.Bundle
import android.view.Window
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        val textView = TextView(this)
        textView.text = "Hello world"
        setContentView(textView)
    }
}
